.github/workflows/integration-test.yml

```diff
@@ -33,9 +33,9 @@ jobs:
         run: |
           until [ $(cat build/generated/launch.complete |wc -l) = 4 ]
           do
-            sleep 10
+            sleep 60
```

Comment:
why do we bump to 60 seconds? Isn't that gonna slow down the test?

Reply:
not going to commit this, just seeing if it eventually builds